systems/data-plan/package/pkg/server/package_test.go

```diff
@@ -0,0 +1,210 @@
+package server
```

Comment:
Good, but we can do better by trying to group these by sub test under the same "behavior under test".